### PR TITLE
pictures modification - backround

### PR DESCRIPTION
**Pull request by Lukaasko**
Some images were unreadable when using dark mode.

### CODE REVIEW

File: content/en/docs/configuration/certificate-authority.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ gRPC API of the Certificate Authority service as defined [here](https://github.c
 | `apis.grpc.keepAlive.maxConnectionAgeGrace` | string | `An additive period after MaxConnectionAge after which the connection will be forcibly closed. 0s means infinity.` | `0s` |
 | `apis.grpc.keepAlive.time` | string | `After a duration of this time if the server doesn't see any activity it pings the client to see if the transport is still alive.` | `2h` |
 | `apis.grpc.keepAlive.timeout` | string | `After having pinged for keepalive check, the client waits for a duration of Timeout and if no activity is seen even after that the connection is closed.` | `20s` |
-| `apis.grpc.tls.caPool` | []string | `File paths to the root certificates in PEM format. The file may contain multiple certificates.` |  `[]` |
+| `apis.grpc.tls.caPool` | string | `File paths to the root certificates in PEM format. The file may contain multiple certificates.` |  `[]` |
```
Comment (jkralik) [inline content/en/docs/configuration/certificate-authority.md:48]: `caPool` supports two types `[]string` and also `string` -> it is better to mention more general case which is `[]string`

pls fix it for all `caPool` cases

File: content/en/docs/configuration/device-provisioning-service.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ CoAP API as specified in the [workflow](./workflow.puml).
 | Property | Type | Description | Default |
 | ---------- | -------- | -------------- | ------- |
 | `apis.coap.address` | string | `Listen specification <host>:<port> for coap client connection.` | `"0.0.0.0:5688"` |
-| `apis.coap.protocols` | []string | `Protocol for coap connection. The supported values are: "tcp", "udp" .` | `["tcp"]` |
+| `apis.coap.protocols` | string | `Protocol for coap connection. The supported values are: "tcp", "udp" .` | `["tcp"]` |
```
Comment (jkralik) [inline content/en/docs/configuration/device-provisioning-service.md:42]: you can allow accept both protocols if you set `["tcp", "udp"]` - so change it to `[]string`

File: content/en/docs/configuration/certificate-authority.md (hunk 1 above)
Comment (Lukaasko) [reply content/en/docs/configuration/certificate-authority.md]: fixed, thanks

File: content/en/docs/configuration/device-provisioning-service.md (hunk 2 above)
Comment (Lukaasko) [reply content/en/docs/configuration/device-provisioning-service.md]: fixed, thanks

**Review by jkralik — APPROVED** [review]

**Pull request merged**